apps/examples/iotivity_simpleserver/simpleServer.c

```diff
@@ -84,7 +84,7 @@
 
 #if defined(CONFIG_ENC28J60) || defined(CONFIG_WICED)
 #define NET_DEVNAME "en"
-#elif defined(CONFIG_ARCH_BOARD_SIDK_S5JT200) || defined(CONFIG_ARCH_BOARD_ARTIK053)
+#elif defined(CONFIG_ARCH_BOARD_SIDK_S5JT200) || defined(CONFIG_ARCH_BOARD_ARTIK053) || defined(CONFIG_ARCH_BOARD_ARTIK055S)
```

Comment:
I think ```defined(CONFIG_ARCH_BOARD_ARTIK05X_FAMILY)``` is better than ```defined(CONFIG_ARCH_BOARD_ARTIK053) || defined(CONFIG_ARCH_BOARD_ARTIK055S)```.
Anyway I don't know why it has board-specific conditional instead of function conditional. It looks not good.

Reply:
I can set CONFIG_ARCH_BOARD_ARTIK05X_FAMILY but I can only test on 55s I guess 53 worked before and we can assume others will.

I plan to make a cleanup patch to introduce CONFIG_NET_DEVNAME that will be set by kconfig to "en" or "wl", would that be ok ?

Reply:
055s and 053s are same on iotitivity so that family will work well.
CONFIG_NET_DEVNAME is better I think.